modules/post/linux/gather/rancher_audit_log_leak.rb

```diff
@@ -0,0 +1,121 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::File
+  include Msf::Auxiliary::Report
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Rancher Audit Log Sensitive Information Leak',
+        'Description' => %q{
+          Rancher versions between 2.6.0-2.6.13, 2.7.0-2.7.9, 2.8.0-2.8.1 inclusive
+          contain a vulnerability where sensitive data is leaked into the audit logs.
+          Rancher Audit Logging is an opt-in feature, only deployments that have it
+          enabled and have AUDIT_LEVEL set to 1 or above are impacted by this issue.
+
+          Tested against rancher 2.6.0.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'h00die', # msf module
+        ],
+        'Platform' => ['linux', 'unix'],
+        'SessionTypes' => ['shell', 'meterpreter'],
+        'References' => [
+          [ 'URL', 'https://github.com/rancher/rancher/security/advisories/GHSA-xfj7-qf8w-2gcr'],
+          [ 'URL', 'https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/enable-api-audit-log#api-audit-log-options'],
+          [ 'CVE', '2023-22649']
+        ],
+        'DisclosureDate' => '2024-02-08',
+        'Notes' => {
+          'Stability' => [],
+          'Reliability' => [],
+          'SideEffects' => []
+        }
+      )
+    )
+    register_advanced_options [
```

Comment:
```suggestion
    # docker install, and default path according to https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/enable-api-audit-log#api-audit-log-options
    register_options [
```
I know this is unlikely to be changed, but in the event it needs to be changed, users should see it.  I'd also be fine if this got left as an advanced option and the warning message contained a reference to changing the advanced option.